.github/workflows/deploy-dev.yml

```diff
@@ -15,14 +15,14 @@ jobs:
     with:
       env: dev
       datahub_helm_version: "0.4.9"
```

Comment:
Can we update the main helm to the most recent version ([`0.4.23`](https://github.com/acryldata/datahub-helm/compare/datahub-0.4.9...datahub-0.4.23))?

Looks like we might need to [add a cron schedule for the `datahubUpgrade.restoreIndices` job](https://github.com/acryldata/datahub-helm/compare/datahub-0.4.9...datahub-0.4.23#diff-2af07fde488e4606b652c73a0fe6450459e28966c085bee6135afb3d26b696eaR343), as it's otherwise going to default as [`"0 0 * * 0"`](https://crontab.guru/#0_0_*_*_0), which would be during downtime.